exporter/awsemfexporter/metric_translator.go

```diff
@@ -48,7 +48,7 @@ type cWMetrics struct {
 type cWMeasurement struct {
 	Namespace  string
 	Dimensions [][]string
-	Metrics    []map[string]string
+	Metrics    []map[string]any
```

Comment:
I don't like the use of `any` here.  Use the type system to prevent invalid data from being represented.  If that means a new type needs to be created then a new type will need to be created.  Rather than `[]map[string]any` this should be something like `[]metricDefinition` with `metricDefinition` defined like this:

```go
type metricDefinition struct {
  Name string
  Unit string
  StorageResolution int
}
```

Reply:
The latest update creates a `cWMetricInfo` struct in the translator to mimic the `metricInfo` struct in grouped_metric.go, replacing the use of `map[string]any` in the translator.